lib/node_modules/@stdlib/stats/base/dists/levy/entropy/benchmark/benchmark.js

```diff
@@ -31,16 +32,23 @@ var entropy = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var len;
 	var mu;
 	var c;
 	var y;
 	var i;
 
+	len = 100;
+	mu = new Float64Array( len );
+	c = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		mu[ i ] = ( randu() * 100.0 ) - 50.0;
```

Comment:
Would you be so kind and update the benchmarks to generate the random values directly from @stdlib/random/base/uniform distribution with the desired range?